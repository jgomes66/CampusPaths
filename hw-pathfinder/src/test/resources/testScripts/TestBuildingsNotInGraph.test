# Simple test to see what happens when the buildings asked for
# is not in the graph

CreateGraph simpleGraph

AddNode simpleGraph v1

FindPath simpleGraph v2 v3