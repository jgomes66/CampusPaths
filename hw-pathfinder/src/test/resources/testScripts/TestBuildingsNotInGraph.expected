# Simple test to see what happens when the buildings asked for
# is not in the graph

created graph simpleGraph

added node v1 to simpleGraph

unknown: v2
unknown: v3